export_test.go

```diff
@@ -256,3 +256,139 @@ func TestExcludeFlag(t *testing.T) {
+	_, err = tx.Exec(ctx, `
+		CREATE TABLE users (
+			id SERIAL PRIMARY KEY,
+			name TEXT,
+			email TEXT,
+			created_at TIMESTAMP DEFAULT NOW(),
+			updated_at TIMESTAMP DEFAULT NOW()
+		);
+		
+		CREATE TABLE posts (
+			id SERIAL PRIMARY KEY,
+			title TEXT,
+			content TEXT,
+			user_id INTEGER REFERENCES users(id),
+			created_at TIMESTAMP DEFAULT NOW(),
+			updated_at TIMESTAMP DEFAULT NOW()
+		);
+		
+		INSERT INTO users (name, email) VALUES ('Alice', 'alice@example.com'), ('Bob', 'bob@example.com');
+		INSERT INTO posts (title, content, user_id) VALUES 
+			('Post 1', 'Content 1', 1), 
+			('Post 2', 'Content 2', 1),
+			('Post 3', 'Content 3', 2);
+	`)
+	require.NoError(t, err)
+
+	// Test 1: Exclude global columns (created_at, updated_at)
+	t.Run("Global column exclusion", func(t *testing.T) {
+		ripoffFile, err := ExportToRipoff(ctx, tx, []string{}, []string{"created_at", "updated_at"})
+		require.NoError(t, err)
+
+		// Verify that no row contains created_at or updated_at columns
+		for rowId, row := range ripoffFile.Rows {
+			_, hasCreatedAt := row["created_at"]
+			_, hasUpdatedAt := row["updated_at"]
+			require.False(t, hasCreatedAt, "Row %s should not have created_at column", rowId)
+			require.False(t, hasUpdatedAt, "Row %s should not have updated_at column", rowId)
+
+			// But should still have other columns
+			tableName := strings.Split(rowId, ":")[0]
+			switch tableName {
+			case "users":
+				_, hasName := row["name"]
+				_, hasEmail := row["email"]
+				require.True(t, hasName, "Row %s should have name column", rowId)
+				require.True(t, hasEmail, "Row %s should have email column", rowId)
+			case "posts":
+				_, hasTitle := row["title"]
+				_, hasContent := row["content"]
+				require.True(t, hasTitle, "Row %s should have title column", rowId)
+				require.True(t, hasContent, "Row %s should have content column", rowId)
+			}
+		}
+	})
+
+	// Test 2: Exclude table-specific column (users.email)
+	t.Run("Table-specific column exclusion", func(t *testing.T) {
+		ripoffFile, err := ExportToRipoff(ctx, tx, []string{}, []string{"users.email"})
+		require.NoError(t, err)
+
+		// Verify that user rows don't have email but post rows still have all columns
+		for rowId, row := range ripoffFile.Rows {
+			tableName := strings.Split(rowId, ":")[0]
+			switch tableName {
+			case "users":
+				_, hasEmail := row["email"]
+				require.False(t, hasEmail, "User row %s should not have email column", rowId)
+				// Should still have other columns
+				_, hasName := row["name"]
+				require.True(t, hasName, "User row %s should have name column", rowId)
+			case "posts":
+				// Posts should have all columns including created_at/updated_at since only users.email was excluded
+				_, hasTitle := row["title"]
+				_, hasCreatedAt := row["created_at"]
+				require.True(t, hasTitle, "Post row %s should have title column", rowId)
+				require.True(t, hasCreatedAt, "Post row %s should have created_at column", rowId)
```

Comment:
Might be slightly more compelling to test using a column shared by both tables, like `users.created_at`

Reply:
Oh yeah I like that a lot better.